src/main/java/me/ryanhamshire/GriefPrevention/DatabaseDataStore.java

```diff
@@ -101,10 +101,13 @@ void initialize() throws Exception
         {
             //ensure the data tables exist
             statement.execute("CREATE TABLE IF NOT EXISTS griefprevention_nextclaimid (nextid INTEGER)");
-            statement.execute("CREATE TABLE IF NOT EXISTS griefprevention_claimdata (id INTEGER, owner VARCHAR(50), lessercorner VARCHAR(100), greatercorner VARCHAR(100), builders TEXT, containers TEXT, accessors TEXT, managers TEXT, inheritnothing BOOLEAN, parentid INTEGER)");
+            statement.execute("CREATE TABLE IF NOT EXISTS griefprevention_claimdata (id INTEGER, owner VARCHAR(50), lessercorner VARCHAR(100), greatercorner VARCHAR(100), builders TEXT, containers TEXT, accessors TEXT, managers TEXT, inheritnothing BOOLEAN, parentid INTEGER, claimExplosions BOOLEAN)");
             statement.execute("CREATE TABLE IF NOT EXISTS griefprevention_playerdata (name VARCHAR(50), lastlogin DATETIME, accruedblocks INTEGER, bonusblocks INTEGER)");
             statement.execute("CREATE TABLE IF NOT EXISTS griefprevention_schemaversion (version INTEGER)");
 
+            //ensure old tables contain columns added in later versions
+            statement.execute("ALTER TABLE griefprevention_claimdata ADD IF NOT EXISTS claimExplosions BOOLEAN");
```

Comment:
SQLite's `ALTER TABLE ADD` syntax doesn't support `IF NOT EXISTS`, this should either be a schema version bump or an attempt to alter the table with the expectation that an exception by this particular statement means the upgrade was already done.

https://www.sqlite.org/lang_altertable.html

Reply:
I see, I'll change that.

Wasn't sure if supporting SQLite was required or just a "nice to have"

The try-catch seems like the least intrusive way to do it.

This was a patch created for my server. As players' builds break when the server is restarted (they rely on withers/tnt)
I figured I'd share it

Reply:
At this point SQLite is "supported" in that we already have some specific changes to make it work, so not breaking it when avoidable is good. GP's database usage in general is horrendous.